config/default.yml

```diff
@@ -1,5 +1,5 @@
 server_port: 9001
-service_map_backend: http://localhost:8000/v2
+service_map_backend: https://api.hel.fi/servicemap/v1
```

Comment:
This commit reverts the last commits changed (added a comment there as well but they are pretty much the same)